packages/dashboards/tests/unit/services/dashboard-data-test.js

```diff
@@ -280,7 +296,9 @@ module('Unit | Service | dashboard data', function(hooks) {
       }
     ];
 
-    const data = service.fetchDataForWidgets(1, widgets);
+    const layout = [{ widgetId: 1 }, { widgetId: 2 }, { widgetId: 3 }];
+
+    const data = service.fetchDataForWidgets(1, widgets, layout);
```

Comment:
Do we have any dashboard acceptance tests that have more widgets than the max concurrency and validates all dashboard widgets load?

Reply:
Added to the existing "dashboard success" test (dashboard has 3 widgets, max concur in test app is 2)